src/InspectorServiceProvider.php

```diff
@@ -95,7 +95,7 @@ public function registerInspectorServiceProviders()
         $this->app->register(GateServiceProvider::class);
 
         // For Laravel >=6
-        if (config('inspector.redis', true) && substr(app()->version(), 0, 1) > 5) {
+        if (config('inspector.redis', true) && version_compare(app()->version(), '6.0.0', '>=')) {
```

Comment:
This is required otherwise it thinks Laravel 10 is Laravel 1. 😬

Reply:
`version_compare` what amazing function :)!?